lms/templates/courseware/program_marketing.html

```diff
@@ -1,3 +1,404 @@
+                  <% index += 1 %>
+                  <div class="profile-item">
+                    <div class="avatar">
+                      <div class="img-holder">
+                        <a href="#instructor-details-mobile-${index}" class="instructor-image">
+                          <img src="${static.url(instructor.get('image'))}" alt="${instructor.get('name')}"/>
+                        </a>
+                      </div>
+                    </div>
+                    <div class="details">
+                      <div class="name">
+                        <a href="#instructor-details-mobile-${index}" class="instructor-label">${instructor.get('name')}</a>
+                      </div>
+                      % if instructor.get('position'):
+                        <div class="dept">${instructor['position'].get('organization_name')}</div>
+                      % endif
+                    </div>
+                    <div class="modal modal-custom" id="instructor-details-mobile-${index}">
+                      <a href="#" class="focusKeeper"></a>
+                      <div class="btn-close modal_close"><a href="#"><i class="fa fa-close"></i></a></div>
+                      <div class="modal-body">
+                        <div class="modal-header">
+                          <div class="instructor-data">
+                            <div class="thumbnail">
+                              <img src="${instructor['image']}" alt="${instructor.get('name')}">
+                            </div>
+                            <h3>
+                              <span class="instructor-name">${instructor.get('name')}</span>
+                              % if instructor.get('position'):
+                                <span>
+                                  ${Text(_('{position} at {organization}')).format(
+                                    position=instructor['position'].get('title'),
+                                    organization=instructor['position'].get('organization_name')
+                                  )}
+                                </span>
+                              % endif
+                            </h3>
+                          </div>
+                        </div>
+                        <div class="instructor-bio">${instructor['bio']}</div>
+                      </div>
+                      <a href="#" class="focusKeeper"></a>
+                    </div>
+                  </div>
+                %endfor
+              </div>
+            </div>
+          </div>
+          <%include file="_program_type_faq.html" args="program_type=program_type" />
+          <div class="accordion-item">
+            <div class="accordion-head">${_("FAQ")}</div>
+            <div class="accordion-content">
+              <ul class="list-unstyled faq-links-list">
+                % for item in faq:
+                  <li class="item">
+                    <a href="#!">${item['question']}</a>
```

Comment:
This should probably be a button as well, but I don't know how easy of a change it could be.